test_api.py

```diff
@@ -114,12 +113,13 @@ def test_bulk(self):
         assert rows2[0][3] == 17
 
         power = list(api.influx.query('select value from power').get_points(measurement='power'))
-        assert power[0]['value'], 1437
-        assert power[1]['value'], 1137.42
+        assert power[0]['value'] == 17
+        assert power[1]['value'] == 16
```

Comment:
@alptilev can you look at the bulk test and verify the expectations? 
from the request data this is fine... can you confirm?